.husky/pre-commit

```diff
@@ -3,4 +3,6 @@
 
 npx lint-staged
 
+npx prettier --ignore-unknown --write ${staged}
```

Comment:
What is setting the variable "staged"?

Reply:
My assumption was that it would be set by lint-staged, which we already had installed for ESLint. I was wrong in that assumption though, so I've now configured lint-staged separately and removed this step from pre-commit.